articles/container-registry/container-registry-oci-artifacts.md

```diff
@@ -65,15 +101,15 @@ Use the `oras push` command to push this text file to your registry. The followi
 
 ```bash
 oras push myregistry.azurecr.io/samples/artifact:1.0 \
-    --manifest-config /dev/null:application/vnd.unknown.config.v1+json \
+    --config /dev/null:application/vnd.unknown.config.v1+json \
```

Comment:
config and +json are only suggested when there's actually a config object, with a specific format:

Please change to:

`--config /dev/null:application/vnd.unknown.v1\`